Basic-Car-Maintenance/Documentation.docc/FirestoreDetails.md

```diff
@@ -0,0 +1,78 @@
+## Firestore Collections
+
+### _alerts_
+
+The alerts collection contains system level alerts that will be visible to all users. The alerts
+will only be displayed to the user once.
+
+| Fields     | Firestore Type | Description                                                                            |
+| ---------- | -------------- | -------------------------------------------------------------------------------------- |
+| actionText | Text           | Text that will appear on the action button                               |
+| actionURL  | Text           | Url that the user will be redirected to when the action button is tapped |
+| emojiIcon  | Text           | The emoji that will appear on the alert message                                        |
+| id         | Text           | Identifies a specific alert                                                            |
+| isOn       | Boolean        | Toggles if the alert is active. Only one alert will be active at a time.             |
+| message    | Text           | Text content of the alert                                                          |
+| title      | Text           | Name given to the alert                                                            |
+
+### maintainance_events
+
+Maintainance events collection contains the individual maintainance events that are associated
+with a specific user.
+
+| Fields    | Firestore Type | Description                                                                               |
+| --------- | -------------- | ----------------------------------------------------------------------------------------- |
+| id        | DocumentID     | Unique maintainance event identifier that will auto-populate your objects id property |
+| userID    | Text           | Identifies the user associated with the maintainance event                                |
+| title     | Text           | Customized name given to the maintainance event                                       |
+| date      | Date           | Date that the maintainance event was created                                          |
```

Comment:
```suggestion
| date      | Date           | Date that the maintenance event was recorded                                          |
```

Going to word it this way because technically it's not the date it's created, but whatever date when someone decides to enter it